.vib/consul/cypress/cypress/support/commands.js

```diff
@@ -1,5 +1,9 @@
-Cypress.on('uncaught:exception', (err, runnable) => {
-  if (err.message.includes('Cannot read properties of undefined')) {
+Cypress.on('uncaught:exception', (err, runnable, promise) => {
+  if (
+    err.message.includes('Cannot read properties of undefined') ||
+    promise ||
+    'is not a function'
+  ) {
```

Comment:
Not sure about this code, as the promise usage is not necessary and the check of `is not a function` should be against the `err.message` var, right? I think we could refactor this to the following:
```suggestion
Cypress.on('uncaught:exception', (err, runnable) => {
  if (err.message.includes('Cannot read properties of undefined') ||
    err.message.includes('is not a function')) {
```

Reply:
Maybe a typo, this was marked as resolved with no further action.